chia/wallet/wallet_spend_bundle.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Tuple
+
+from chia_rs import AugSchemeMPL, G2Element
+
+from chia.consensus.default_constants import DEFAULT_CONSTANTS
+from chia.types.coin_spend import CoinSpend
+from chia.types.spend_bundle import SpendBundle, T_SpendBundle
+from chia.wallet.util.debug_spend_bundle import debug_spend_bundle
+
+
+class WalletSpendBundle(SpendBundle):
+
+    @staticmethod
+    def from_bytes(bytes: bytes) -> WalletSpendBundle:
+        sb = SpendBundle.from_bytes(bytes)
+        return WalletSpendBundle(sb.coin_spends, sb.aggregated_signature)
+
+    @staticmethod
+    def parse_rust(blob: bytes, flag: bool = False) -> Tuple[WalletSpendBundle, int]:
+        bundle, advance = super(WalletSpendBundle, WalletSpendBundle).parse_rust(blob)
+        return WalletSpendBundle(bundle.coin_spends, bundle.aggregated_signature), advance
```

Comment:
These being static methods will mean that if we ever subclass this, these methods won't work (they will return the parent).  Is that intentional?

Reply:
Nope, I turned them into class methods as a result. Thanks.